modules/apps/portal/portal-log4j/src/main/java/com/liferay/portal/log4j/internal/CompanyWebIdLogContext.java

```diff
@@ -34,8 +36,15 @@ public Map<String, String> getContext(String logName) {
 			return Collections.emptyMap();
 		}
 
-		return Collections.singletonMap(
-			"webId", PortalInstances.getWebId(companyId));
+		String webId = PortalInstances.getWebId(companyId);
+
+		if (webId == null) {
+			Company company = CompanyLocalServiceUtil.fetchCompany(companyId);
```

Comment:
@yuhai Could you temporally enable your property and run CI again? Since the log output can happen during adding company, I wonder if this query may trigger any Hibernate auto_flush....